tests/test_benchmark.py

```diff
@@ -0,0 +1,69 @@
+import pyhf
+from pyhf.tensor.pytorch_backend import pytorch_backend
+from pyhf.tensor.numpy_backend import numpy_backend
+from pyhf.tensor.tensorflow_backend import tensorflow_backend
+from pyhf.tensor.mxnet_backend import mxnet_backend
+from pyhf.simplemodels import hepdata_like
+import tensorflow as tf
+import pytest
+
+
+def generate_source(n_bins):
+    """
+    Create the source structure for the given number of bins.
+    Currently the data that are being put in is all the same
+    but in the future it should be generated pseudodata.
+    """
+    binning = [n_bins, -0.5, 1.5]
+    data = [120.0]
```

Comment:
This could be implemented more cleanly. e.g. you could do something like

```
data = [120.] * n_bins
bkg = [100.] * n_bins
bkgerr = [10.0] * n_bins
sig = [30.0] * n_bins
```

Reply:
maybe we can have random varations but with a fixed see, so that it's reproducible